MobiFlight/Joysticks/JoystickManager.cs

```diff
@@ -121,7 +128,9 @@ public void SetHandle(IntPtr handle)
         public void Connect()
         {
             var di = new SharpDX.DirectInput.DirectInput();
-            joysticks?.Clear();
+            Joysticks?.Clear();
+            ExcludedJoysticks?.Clear();
+            string[] excludedJoysticks = Properties.Settings.Default.ExcludedJoysticks.Split('§');            
```

Comment:
we never used § as a delimeter yet. You can actually serialize the data structure like I did in MainForm for the autoload-configs... i think that is way nicer.

see MainForm > RestoreAutoLoadConfig & SaveAutoLoadConfig

Reply:
yeah that's nicer. Done.